lib/orbit/ocs/stream/pipeline.ex

```diff
@@ -39,16 +42,24 @@ defmodule Orbit.Ocs.Stream.Pipeline do
 
     event_records =
       records
-      |> Enum.flat_map(&parse_records/1)
+      |> Stream.flat_map(&parse_records/1)
       # |> update_ocs_sequence_monitor
-      |> Enum.map(&ocs_message/1)
+      |> Stream.filter(&ocs_message?/1)
+      |> Enum.map(&unwrap_ocs_data/1)
```

Comment:
suggestion: probably favor `Enum.filter` here since the `Enum.map` that immediately follows is not lazy?